src/foam/core/reflow/FromCsvRefines.js

```diff
@@ -0,0 +1,69 @@
+foam.CLASS({
+  package: 'foam.lib.csv',
+  name: 'PropertyFromCSV',
+  refines: 'foam.lang.Property',
+  properties: [
+    {
+      class: 'Function',
+      name: 'fromCSV',
+      value: function(str) {
+        return this.fromString(str);
+      }
+    }
+  ]
+});
+foam.CLASS({
+  name: 'IntFromCSVRefines',
+  refines: 'foam.lang.Int',
+  properties: [
+    {
+      class: 'Function',
+      name: 'fromCSV',
+      value: function(str) {
+        return this.fromString(str?.replace(/,/g, ''));
+      }
+    }
+  ]
+});
+
+foam.CLASS({
+  name: 'FloatFromCSVRefines',
+  refines: 'foam.lang.Float',
+  properties: [
+    {
+      class: 'Function',
+      name: 'fromCSV',
+      value: function(str) {
+        return this.fromString(str?.replace(/,/g, ''));
+      }
+    }
+  ]
+});
+
+foam.CLASS({
+  name: 'LongFromCSVRefines',
```

Comment:
Long extends Int so you don't need to do it here also. Actually, Float extends Int also, so you should only have to do it in Int.